cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -102,24 +103,27 @@ func (s *TunnelServer) Start() {
 	s.installDefaultHandler()
 	data, err := ioutil.ReadFile(config.Config.TLSTunnelCAFile)
 	if err != nil {
-		klog.Fatalf("Read tls tunnel ca file error %v", err)
-		return
+		data = hubconfig.Config.Ca
+		klog.Info("Succeeded in loading TLSTunnelCAFile from the secret")
 	}
 	pool := x509.NewCertPool()
-	pool.AppendCertsFromPEM(data)
+	pool.AppendCertsFromPEM(pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: data}))
 
 	cert, err := ioutil.ReadFile(config.Config.TLSTunnelCertFile)
```

Comment:
Do we need independent certs for the tunnel? @kadisi

Reply:
yes. 

If k8s cluster is installed by kubeadm, when kubeapiserver accesses kubelet, the certificate of kubelet will not be validated by default.
However, kube-apiserver also has options configured to verify the validity of kubelet certificates.In the future we should need the ca certificate of kube-apiserver to create the cloudstream certificate.